core/base/src/TApplication.cxx

```diff
@@ -1060,9 +1061,11 @@ void TApplication::OpenGitHubIssue(const TString &type)
     } else if (type == "feature") {
     OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=new+feature&template=feature_request.yml\"");
 } else {
-       Warning("OpenGitHubIssue", "cannot find \"%s\" as type for a GitHub issue\n"
-               "Available types are 'bug', 'feature' or 'improvement'.", type.Data());
+       Warning("OpenGitHubIssue", "Cannot find GitHub issue type \"%s\".\n"
+               "Available types are 'bug', 'feature' and 'improvement'.", type.Data());
     }
+
+   Info("OpenGitHubIssue", "A new tab should have opened in your browser.");
```

Comment:
This should not be printed if the Warning on 1064 was printed

Reply:
Maybe it could be moved to the OpenInBrowser function?

Reply:
of course, thank you, moved it up (I prefer keeping the logging in the outer layer)